mozartdata/transforms/bridge/orders.sql

```diff
@@ -345,6 +357,22 @@ SELECT orders.order_id_edw
 	 , aggregates.amount_paid_total
 	 , aggregates.gross_profit_estimate
 	 , aggregates.cost_estimate
+    , case
+        when
+            aftership_return_orders.original_order_id_edw is not null
+        then
+            true
+        else
+            false
+    end as has_aftership_rma -- indicates if the order has an aftership rma associated with it
+    , case
+        when
+            aftership_exchange_orders.rma_exchange_order_id_edw is not null
+        then
+            true
+        else
+            false
+    end as is_aftership_generated -- indicates if the order was created by Aftership as part of an rma
```

Comment:
is this the right name change?
```suggestion
    , case
        when
            aftership_exchange_orders.rma_exchange_order_id_edw is not null
        then
            true
        else
            false
    end as is_aftership_exchange -- indicates if the order was created by Aftership as part of an rma
```

Reply:
@goodrsamwise - did you make this change? or do you propose something else?

Reply:
totally didn't see this. updating in the push